addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/common/HomematicConfig.java

```diff
@@ -51,6 +51,7 @@
     private int timeout = 15;
     private int installModeDuration = DEFAULT_INSTALL_MODE_DURATION;
     private long discoveryTimeToLive = -1;
+    private boolean unpairOnDeletion = false;
```

Comment:
Is this redundant initialization needed in this case? For objects `null` is default, booleans have there default `false` and numbers the default `0`. Actually they also generate more bytecode. I.e. slightly more storage and slightly less performance, see [Java Practices](http://www.javapractices.com/topic/TopicAction.do?Id=14).

Reply:
I put the initialization there to make it explicit that `false` is the default value, but I did not consider a potential performance impact this might have. I think a good compromise might be to remove the initialization and add a comment that still makes the intention clear.